tiingo/api.py

```diff
@@ -128,6 +136,36 @@ def get_ticker_metadata(self, ticker, fmt='json'):
         elif fmt == 'object':
             return dict_to_object(data, "Ticker")
 
+    def _invalid_frequency(self, frequency):
+        """
+        Check to see that frequency was specified correctly
+        :param frequency (string): frequency string
+        :return (boolean):
+        """
+        daily_freqs = ['daily', 'weekly', 'monthly', 'annually']
+
+        if frequency.lower() in daily_freqs or re.match(self._frequency_pattern, frequency):
```

Comment:
Since the value being tested is already a boolean, we may not need to have an `if` tree. Could we try something like this?

```python
is_valid_frequency = frequency.lower() in daily_freqs or re.match(self._frequency_pattern, frequency)
return not is_valid_frequency
```

Reply:
Ok, that's good - changed.